tikv-client/src/main/java/com/pingcap/tikv/TiConfiguration.java

```diff
@@ -71,6 +74,14 @@ public static TiConfiguration createDefault(String pdAddrsStr) {
     return PDUtils.addrsToUrls(addrs);
   }
 
+  public DateTimeZone getLocalTimeZone() {
+    return localTimeZone;
+  }
+
+  public boolean checkLocalTimeZone() {
```

Comment:
any reason why these functions should be in `TiConfiguration`?

Reply:
we want to add `driver's timezone` to Spark's Task, then serialize and send to spark executor. 

`TiConfiguration` is already added to Spark's Task.

Reply:
I mean, there are many things that are sent to Spark executors, is there a better choice?

Reply:
why many things?

Reply:
add a `DateTimeZone` for each task

Reply:
removed